rero_ils/modules/operation_logs/cli.py

```diff
@@ -87,7 +87,7 @@ def dump_operation_logs(outfile_name, year):
 
     index_count = 0
     outfile = JsonWriter(outfile_name)
-    with click.progressbar(search.scan()) as bar:
+    with click.progressbar(search.scan(), length=search.count()) as bar:
```

Comment:
Could the commit message be more precise? Add the search count to the progress bar or something?